### DISCUSSION TITLE
csv-file for MF=Vaillant;ID=BAI00;SW=0703;HW=7401 will not be loaded after update to 23.3

### DISCUSSION THREAD

**Discussion by Nick-s77** [john30/ebusd#1199] [Q&A]
Since the update to v.23.3 ebusd could not find following csv-files anymore:
"MF=Vaillant;ID=BAI00;SW=0703;HW=7401", loaded "vaillant/bai.308523.inc" ([HW=7401]), "vaillant/08.bai.csv" 

With version 23.2 I got:

ebusctl info :
version: ebusd 23.2.23.2
device: 192.168.227.143:9999, enhanced, firmware 1.1[4106].1[4106]
access: *
signal: acquired
symbol rate: 22
max symbol rate: 103
min arbitration micros: 7
max arbitration micros: 1128
min symbol latency: 12
max symbol latency: 43
scan: finished
reconnects: 0
masters: 4
messages: 227
conditional: 3
poll: 0
update: 10
address 00: master #1
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0703;HW=7401", loaded "vaillant/bai.308523.inc" ([HW=7401]), "vaillant/08.bai.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=70000;SW=0510;HW=6403", loaded "vaillant/15.700.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd

Any ideas, why this device cannot be found any more with newest version?

**Comment by sfoglia** [comment]
` $ docker exec -it  ebusd ebusctl info`
```
version: ebusd 23.3.23.3-37-g741860f
device: 10.1.1.33:9999, TCP, enhanced, firmware 1.1[4106].1[4106]
signal: acquired
symbol rate: 23
max symbol rate: 60
min arbitration micros: 8
max arbitration micros: 12
min symbol latency: 11
max symbol latency: 144
scan: finished
reconnects: 1
masters: 2
messages: 224
conditional: 3
poll: 0
update: 10
address 03: master #11
address 04: slave #25, ebusd (answering)
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0104;HW=7803", loaded "vaillant/bai.308523.inc", "vaillant/08.bai.csv"
address ff: master #25, ebusd (answering)
```

**Reply by Nick-s77** [reply]
thanks for the response !!!

but this is another hardware 
mine is: BAI00;SW=0703;HW=7401"
yours is: BAI00;SW=0104;HW=7803

and why is mine no longer assigned to "vaillant/bai.308523.inc", "vaillant/08.bai.csv" ???

**Comment by JonesGW-MGD** [comment]
Can you share the logs at startup for 23.3 then to compare?  The 08.bai.csv which determines which .inc file to load hasn't changed for 5 months and @sfoglia's post shows that bai.08.csv and bai.308523.inc clearly work at v23.3.

The hardware difference isn't really relevant - their bai.30823.inc loads because there is no match in bai.08.csv and loads this default file at line 30.  Yours does match (hardware fallback at line 28) but the same default file is specified so should load the same file.

I'd presume that you have some issue with paths or file locations and that on startup there will be some clues to the problem.  What more can you tell us about what --configpath you have, etc. etc.
